Configurations.md

```diff
@@ -1288,6 +1288,14 @@ If you want to ignore every file under the directory where you put your rustfmt.
 ignore = ["/"]
 ```
 
+## `ignore_missing_submod`
+
+Ignore missing submodule error.
```

Comment:
I think we need to elaborate on this description. I think we should explain that normally missing modules will lead to module not found errors and rustfmt won't format anything.

Reply:
Fixed. Most of the messages you wrote are used, but I added "By default" at the beginning of the sentence to clarify the behavior by default.